docs/AccessScalarProducts.md

```diff
@@ -0,0 +1,185 @@
+
+* Example (if you deploy Scalar products on the `ns-scalar` namespace)
+  * ScalarDB Server
+    ```console
+    scalardb-envoy.ns-scalar.svc.cluster.local
+    ```
+  * ScalarDL Ledger
+    ```console
+    scalardl-ledger-envoy.ns-scalar.svc.cluster.local
+    ```
+  * ScalarDL Auditor
+    ```console
+    scalardl-auditor-envoy.ns-scalar.svc.cluster.local
+    ```
+
+If you use the service resource of Kubernetes, you need to set the above FQDN in the properties file for the application (client) as follows.
+
+* Client properties file for ScalarDB Server
+  ```properties
+  scalar.db.contact_points=<helm release name>-envoy.<namespace>.svc.cluster.local
+  scalar.db.contact_port=60051
+  scalar.db.storage=grpc
+  scalar.db.transaction_manager=grpc
+  ```
+* Client properties file for ScalarDL Ledger
+  ```properties
+  scalar.dl.client.server.host=<helm release name>-envoy.<namespace>.svc.cluster.local
+  scalar.dl.ledger.server.port=50051
+  scalar.dl.ledger.server.privileged_port=50052
+  ```
+* Client properties file for ScalarDL Ledger and ScalarDL Auditor (Auditor mode)
+  ```properties
+  # Ledger
+  scalar.dl.client.server.host=<helm release name>-envoy.<namespace>.svc.cluster.local
+  scalar.dl.ledger.server.port=50051
+  scalar.dl.ledger.server.privileged_port=50052
+  
+  # Auditor
+  scalar.dl.client.auditor.enabled=true
+  scalar.dl.client.auditor.host=<helm release name>-envoy.<namespace>.svc.cluster.local
+  scalar.dl.auditor.server.port=40051
+  scalar.dl.auditor.server.privileged_port=40052
+  ```
+
+## Access Scalar products via Load Balancers from outside of the Kubernetes cluster
+
+If you deploy your application (client) on another environment than the Kubernetes cluster for Scalar products (e.g., you deploy your application on another  Kubernetes cluster, container platform, or server), the application can access Scalar products using a Load Balancer provided by each cloud service.
+
+You can create the Load Balancer to set `envoy.service.type` to `LoadBalancer` in your custom values file. When you create it, you can access Scalar Envoy (A service resource of Kubernetes) via created Load Balancer. You can also configure the Load Balancer configurations using annotations. Please refer to the following document for more details on how to configure your custom values file.
+
+* [Service configurations](https://github.com/scalar-labs/helm-charts/blob/main/docs/configure-custom-values-envoy.md#service-configurations)
+
+If you use the Load Balancer, you need to set the FQDN or IP address of the Load Balancer in the properties file for the application (client) as follows.
```

Comment:
```suggestion
You can create a load balancer by setting `envoy.service.type` to `LoadBalancer` in your custom values file. After configuring the custom values file, you can use Scalar Envoy as a Kubernetes service resource through the load balancer that you created. You can also configure the load balancer configurations by using annotations.

For more details on how to configure your custom values file, see [Service configurations](https://github.com/scalar-labs/helm-charts/blob/main/docs/configure-custom-values-envoy.md#service-configurations).

When using a load balancer, you must set the FQDN or IP address of the load balancer in the properties file for the application (client) as follows.
```

Reply:
Sorry for a very minor thing, but strictly the Scalar Envoy doesn't equal Kubernetes service. The request flow is the following.
```
[Client] ---> [Load balancer] ---> [Kubernetes service (k8s service)] ---> [Scalar Envoy (k8s pod)] ---> [ScalarDB/DL (k8s pod)]
```

So, I think the following explanation is better.
```
After configuring the custom values file, you can use Scalar Envoy through Kubernetes service resource through the load balancer that you created.
```

However, this sentence uses several `through` like `use AAA through BBB through CCC`.
Is this OK from the English/documentation perspective?
If no, do you have any idea to represent the above request flow in the document?

Reply:
> I think we should avoid using "through" more than once in this sentence. How about we change the last "through" to "by using"?

Looks good to me! Thank you!
I will refer to this when writing documentation in the future.